src/models/service/transactions/details.rs

```diff
@@ -0,0 +1,56 @@
+use serde::Serialize;
+use super::*;
+use crate::models::commons::{Operation, DataDecoded};
+
+#[derive(Serialize, Debug)]
+#[serde(rename_all = "camelCase")]
+pub struct TransactionDetails {
+    pub executed_at: Option<i64>,
+    pub submitted_at: Option<i64>,
+    pub tx_status: TransactionStatus,
+    pub tx_info: TransactionInfo,
+    pub tx_data: Option<TransactionData>,
+    pub detailed_execution_info: Option<DetailedExecutionInfo>,
+    pub tx_hash: Option<String>,
+}
+
+#[derive(Serialize, Debug)]
+#[serde(tag = "type", rename_all = "SCREAMING_SNAKE_CASE")]
+pub enum DetailedExecutionInfo {
+    Multisig(MultisigExecutionDetails),
+    Module(ModuleExecutionDetails),
+    Ether(EtherTransfer),
+}
+
+#[derive(Serialize, Debug)]
+#[serde(rename_all = "camelCase")]
+pub struct MultisigExecutionDetails {
+    pub nonce: u64,
+    pub safe_tx_hash: String,
+    pub signers: Vec<String>,
+    pub confirmations_required: u64,
+    pub confirmations: Vec<MultisigConfirmation>,
```

Comment:
Don't we have already an implicit list of `signers` within `confirmations`, just by doing `confirmations.map { it.signer }` ? Feels a bit duplicated.

Reply:
`signers` are all potential signers (aka `owners` ... we can rename it to that). `confirmations` refers to already submitted confirmations.

Reply:
ah I see, `confirmations` is a list of submitted signatures.